receiver/prometheusreceiver/config.go

```diff
@@ -51,8 +51,17 @@ var _ config.CustomUnmarshable = (*Config)(nil)
 
 // Validate checks the receiver configuration is valid
 func (cfg *Config) Validate() error {
-	if cfg.PrometheusConfig != nil && len(cfg.PrometheusConfig.ScrapeConfigs) == 0 {
-		return errNilScrapeConfig
+	if cfg.PrometheusConfig != nil {
+		if len(cfg.PrometheusConfig.ScrapeConfigs) == 0 {
```

Comment:
Can you add a TODO and link to a tracking issue so we can ensure it will be addressed in the future?

Reply:
done